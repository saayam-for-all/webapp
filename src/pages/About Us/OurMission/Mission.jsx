export function Mission() {
  return (
    <section className="flex justify-center items-center py-10 bg-white">
      <div className="w-[1100px] h-[565px] rounded-[30px] p-6 shadow-md border border-blue-200 flex flex-col justify-between">
        {/* Header Row: Our Mission + Paragraph in single row */}
        <div className="flex mb-6">
          <h2
            className="text-2xl font-bold mr-4 whitespace-nowrap"
            style={{ fontFamily: "Josefin Sans" }}
          >
            Our Mission
          </h2>
          <p
            className="text-gray-600 text-base leading-relaxed"
            style={{ fontFamily: "Josefin Sans" }}
          >
            Our mission is to provide timely and targeted assistance to those
            facing challenges in their lives, fostering a culture of compassion,
            solidarity, and service to humanity.
          </p>
        </div>

        {/* Image Row */}
        <div className="flex justify-between gap-4">
          <img
            src="https://cdn.builder.io/api/v1/image/assets/TEMP/587ed268b2153793efab648656556a802eb62ad9"
            alt="Mission image 1"
            className="w-1/3 h-[350px] object-cover rounded-[20px]"
          />
          <img
            src="https://cdn.builder.io/api/v1/image/assets/TEMP/67c999ae47a8163906957ca47ca9b1d509b125ac"
            alt="Mission image 2"
            className="w-1/3 h-[350px] object-cover rounded-[20px]"
          />
          <img
            src="https://cdn.builder.io/api/v1/image/assets/TEMP/4a2a77f3953d04155e4de9a3f393c7266b008dae"
            alt="Mission image 3"
            className="w-1/3 h-[350px] object-cover rounded-[20px]"
          />
        </div>
      </div>
    </section>
  );
}
