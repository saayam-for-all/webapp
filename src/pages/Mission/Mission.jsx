import React from "react";
import { useTranslation } from "react-i18next";
import "./Mission.css";
import MISSIONIMG from "../../assets/mission.png";

const MissionVision = () => {
  const { t } = useTranslation();
  return (
    <div className="px-5 md:px-10 lg:px-20 mt-6 mb-6">
      <div>
        <h1 className="text-3xl font-semibold text-center">{t("MISSION")}</h1>
        <div className="flex flex-col md:flex-col lg:flex-row max-w-5xl mx-auto mt-5">
          <div className="flex-shrink-0">
            <img src={MISSIONIMG} alt="mission" className="w-[35rem] h-80" />
          </div>
          <p className="text-lg px-5 text-justify">
            {t("EMPOWER_INDIVIDUALS")}
          </p>
        </div>
      </div>
    </div>
  );
};

export default MissionVision;
