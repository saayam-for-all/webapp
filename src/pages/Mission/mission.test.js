import { render } from "@testing-library/react";
import Mission from "./Mission";

describe("Mission", () => {
  it("renders correctly", () => {
    const tree = render(<Mission />);
    expect(tree).toMatchSnapshot();
  });
});
