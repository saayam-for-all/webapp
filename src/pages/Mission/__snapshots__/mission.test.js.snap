// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mission renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="px-20 mt-6"
      >
        <div>
          <h1
            class="text-2xl font-semibold text-center"
          >
            mockTranslate(MISSION)
          </h1>
          <div
            class="flex"
          >
            <div>
              <img
                alt="mission"
                class="w-[300rem] h-80 mt-5"
                src=""
              />
            </div>
            <p
              class="mt-5 text-lg px-5"
            >
              mockTranslate(EMPOWER_INDIVIDUALS)
            </p>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="px-20 mt-6"
    >
      <div>
        <h1
          class="text-2xl font-semibold text-center"
        >
          mockTranslate(MISSION)
        </h1>
        <div
          class="flex"
        >
          <div>
            <img
              alt="mission"
              class="w-[300rem] h-80 mt-5"
              src=""
            />
          </div>
          <p
            class="mt-5 text-lg px-5"
          >
            mockTranslate(EMPOWER_INDIVIDUALS)
          </p>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
